datatable/extras/aggregate.py

```diff
@@ -0,0 +1,14 @@
+    #!/usr/bin/env python3
```

Comment:
no spaces before the `#!`

Reply:
Yes, another bug of my editor. Fixed now. Thanks.